test/Tests/src/Data/Time/Duration_Spec.enso

```diff
@@ -3,99 +3,88 @@ from Standard.Base import all
-            interval = 4.years + 8.milliseconds
-            interval.to_vector . should_equal [4, 0, 0, 0, 0, 0, 8000000]
+foreign js js_date year month_index=0 day=1 = """
+    return new Date(year, month_index, day)
 
-        Test.specify "should substract seconds from months" <|
-            interval = 8.months - 8.seconds
-            interval.to_vector . should_equal [0, 8, 0, 0, 0, -8, 0]
-
-        Test.specify "should subtract years from hours" <|
-            interval = 2.hours - 11.years
-            interval.to_vector . should_equal [-11, 0, 0, 2, 0, 0, 0]
+spec =
+    Test.group "Duration" <|
 
-        Test.specify "should support mixed operators" <|
-            interval = 2.hours + 12.seconds - 11.years
-            interval.to_vector . should_equal [-11, 0, 0, 2, 0, 12, 0]
+        Test.specify "should create interval seconds" <|
+            duration = 5.seconds
+            duration.seconds . should_equal 5
 
         Test.specify "should create interval between two points in time" <|
-            time1 = Date_Time.new 2001 1 2
-            time2 = Date_Time.new 2001 2 1
-            interval = Duration.between time1 time2
-            interval.to_vector . should_equal [0, 0, 0, 720, 0, 0, 0]
-
-        Test.specify "should check if time based" <|
-            interval = 10.hours
-            interval.is_date . should_be_false
-            interval.is_time . should_be_true
-
-        Test.specify "should check if date based" <|
-            interval = 10.years
-            interval.is_date . should_be_true
-            interval.is_time . should_be_false
-
-        Test.specify "should check if mixed based" <|
-            interval = 10.years + 3.hours
-            interval.is_date . should_be_true
-            interval.is_time . should_be_true
+            Duration.between (Date_Time.new 2001 1 1 3) (Date_Time.new 2001 1 1 10) . should_equal (Duration.new hours=7)
+            Duration.between (Date_Time.new 2001 1 1) (Date_Time.new 2001 1 7) . should_equal (Duration.new hours=(6 * 24))
+            Duration.between (Date_Time.new 2001 1 1 13) (Date_Time.new 2001 1 7 16) . should_equal (Duration.new hours=(3 + 6 * 24))
 
         Test.specify "should check if empty" <|
             interval = 0.seconds
             interval.is_empty . should_be_true
 
         Test.specify "should normalize periods" <|
-            interval = 12.months
-            interval.to_vector . should_equal [1, 0, 0, 0, 0, 0, 0]
+            duration = 60.seconds
+            duration.minutes . should_equal 1
+            duration.hours . should_equal 0
 
         Test.specify "should normalize addition" <|
-            interval = 11.months + 1.month
-            interval.to_vector . should_equal [1, 0, 0, 0, 0, 0, 0]
+            duration = 11.hours + 1.hours
+            duration.hours . should_equal 12
 
         Test.specify "should normalize subtraction" <|
-            interval = 13.months - 1.month
```

Comment:
Removing these tests seems suspicious to me. I believe `month` or `year` is a case that requires `Period` and not `Duration` - does it mean `Period` will no longer be supported, if this PR is integrated? CCing @radeusgd and @jdunkerley .

Reply:
I believe they've been just moved around, right @Akirathan ?